src/utils/client.js

```diff
@@ -177,7 +182,17 @@ export default class Client {
     steps.forEach((step) => {
       step.servar_fields.forEach((field) => {
         const val = getDefaultFieldValue(field);
-        values[field.servar.key] = field.servar.repeated ? [val] : val;
+        // This can't be done when the canvas is hydrated with initial value
```

Comment:
this comment confused me a little because i thought the canvas hydration logic was fairly detached from this logic. the hydration logic hydrates a canvas, while this logic formats initialValue data URLs to be set to fieldValues

Reply:
hm ok yeah, on second thought I think this comment is more confusing than helpful. are you ok with me just removing it?

Reply:
👍